modules/data-mover-for-csi-snapshots.adoc

```diff
@@ -0,0 +1,211 @@
+  # The repository encryption key
+  RESTIC_PASSWORD: <secure-restic-password>
+----
++
+[NOTE]
+====
+By default, the operator looks for a secret named `dm-credential`. If you are using a different name, you need to specify the name via Data Protection Application (DPA) CR using `dpa.spec.features.dataMover.credentialName`.
+====
+. Create a DPA similar to the following example. Note that `defaultPlugins` includes CSI.
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-sample
+  namespace: openshift-adp
+spec:
+  features:
+    dataMover:
+      enable: true
+      credentialName: <secret-name> <1>
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: <bucket-name>
+          prefix: <bucket-prefix>
+        provider: aws
+  configuration:
+    restic:
+      enable: <true or false>
+    velero:
+      defaultPlugins:
+        - openshift
+        - aws
+        - csi
+----
+<1> Add the Restic secret name from the previous step. If this is not done,  the default secret name `dm-credential` is used.
++
+The OADP operator installs two custom resource definitions (CRDs), `VolumeSnapshotBackup` and `VolumeSnapshotRestore`.
++
+.Example `VolumeSnapshotBackup` CR
++
+[source,yaml]
+----
+apiVersion: datamover.oadp.openshift.io/v1alpha1
+kind: VolumeSnapshotBackup
+metadata:
+  name: <vsb-name>
+  namespace: <namespace-name> <1>
+spec:
+  volumeSnapshotContent:
+    name: <snapcontent-name>
+  protectedNamespace: <adp-namespace>
+  resticSecretRef:
+    name: <restic-secret-name>
+----
+<1> Specify the namespace where the volume snapshot resides.
++
+.Example `VolumeSnapshotRestore` CR
++
+[source,yaml]
+----
+apiVersion: datamover.oadp.openshift.io/v1alpha1
+kind: VolumeSnapshotRestore
+metadata:
+  name: <vsr-name>
+  namespace: <namespace-name> <1>
+spec:
+  protectedNamespace: <protected-ns> <2>
+  resticSecretRef:
+    name: <restic-secret-name>
+  volumeSnapshotMoverBackupRef:
+    sourcePVCData:
+      name: <source-pvc-name>
+      size: <source-pvc-size>
+    resticrepository: <your-restic-repo>
+    volumeSnapshotClassName: <vsclass-name>
+----
+<1> Specify the namespace where the volume snapshot resides.
+<2> Specify the namespace where the operator is installed. The default is `openshift-adp`.
+
+. To back up a snapshot, do the following:
+
+.. Create a backup CR:
+
++
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: <backup-name>
+  namespace: <protected-ns> <1>
+spec:
+  includedNamespaces:
+  - <app-ns>
+  storageLocation: velero-sample-1
+----
+<1> Specify the namespace where the operator is installed. The default is `openshift-adp`.
++
+.. Wait several minutes and check whether the VolumeSnapshotBackup CR status is `Completed`:
```

Comment:
1."Several" is vague. Maybe "Wait up to five minutes..." or something like that. 
2. Are there backticks around  VolumeSnapshotBackup? there should be.
3. What do they do if the status is anything other than `Completed`?